syntax = "proto3";

package mcproto;

message CpuLoadInfo
{
    uint32 cpuLoad = 1;
}
